data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -169,430 +143,221 @@ def get_table_dates_info(project_id, dataset_id, client):
-                    dataset=date_row.dataset_id,
-                    table=date_row.table,
-                    pid=pid,
-                    deactivated_pids_project=pids_project_id,
-                    deactivated_pids_dataset=pids_dataset_id,
-                    deactivated_pids_table=pids_table,
-                    end_date_column=date_row.end_date_column,
-                    start_date_column=date_row.start_date_column)
-            else:
-                sandbox_query = SANDBOX_QUERY_DATE.render(
-                    project=date_row.project_id,
-                    sandbox_dataset=sandbox_dataset,
-                    dataset=date_row.dataset_id,
-                    table=date_row.table,
-                    pid=pid,
-                    deactivated_pids_project=pids_project_id,
-                    deactivated_pids_dataset=pids_dataset_id,
-                    deactivated_pids_table=pids_table,
-                    date_column=date_row.date_column)
-                clean_query = CLEAN_QUERY_DATE.render(
-                    project=date_row.project_id,
-                    dataset=date_row.dataset_id,
-                    table=date_row.table,
-                    pid=pid,
-                    deactivated_pids_project=pids_project_id,
-                    deactivated_pids_dataset=pids_dataset_id,
-                    deactivated_pids_table=pids_table,
-                    date_column=date_row.date_column)
-            queries_list.append({
-                clean_consts.QUERY:
-                    sandbox_query,
-                clean_consts.DESTINATION:
-                    date_row.project_id + '.' + sandbox_dataset + '.' +
-                    (ticket_number + '_' + date_row.table),
-                clean_consts.DESTINATION_DATASET:
-                    date_row.dataset_id,
-                clean_consts.DESTINATION_TABLE:
-                    date_row.table,
-                clean_consts.DISPOSITION:
-                    bq_consts.WRITE_APPEND,
-                'type':
-                    'sandbox'
-            })
-            queries_list.append({
-                clean_consts.QUERY:
-                    clean_query,
-                clean_consts.DESTINATION:
-                    date_row.project_id + '.' + date_row.dataset_id + '.' +
-                    date_row.table,
-                clean_consts.DESTINATION_DATASET:
-                    date_row.dataset_id,
-                clean_consts.DESTINATION_TABLE:
-                    date_row.table,
-                clean_consts.DISPOSITION:
-                    bq_consts.WRITE_TRUNCATE,
-                'type':
-                    'retraction'
-            })
-    LOGGER.info(
-        f"Query list complete, retracting ehr deactivated PIDS from the following datasets: "
-        f"{dataset_list}")
-    return queries_list
-
-
-def run_queries(queries, client):
+        sandbox_table = f"{'_'.join(ISSUE_NUMBERS).lower().replace('-', '_')}_{table}"
```

Comment:
After discussion, `ISSUE_NUMBERS` was supposed to be used for reporting purposes if we were given a list of deactivated pids, to indicate the ticket for such request. However, deactivation is being performed at the Unioned stage  of CDR rather than via ticket requests, obviating the need for a ticket number argument (which has caused issues in the past).
This does not diminish the purpose of `Ticket Number` at the Unioned stage of the CDR. A log statement listing the ticket number generating the Unioned Stage of the CDR would be informative, and can be extended to other stages.